pkg/controller/servicemesh/webhooks/validation/controlplane.go

```diff
@@ -46,7 +46,7 @@ func (v *ControlPlaneValidator) Handle(ctx context.Context, req admission.Reques
 		return admission.Allowed("")
 	}
 
-	if req.Namespace == common.GetOperatorNamespace() {
+	if !common.Config.OLM.SplitModeEnabled && req.Namespace == common.GetOperatorNamespace() {
```

Comment:
I'm not sure about the `SplitModeEnabled` field. Can't we detect whether a request is coming from a remote cluster and decide based on that?

Reply:
I think we can use the `ControlPlaneMode` flag here because that should be the only case when we want this to be turned off

Reply:
So if split mode is enabled, you can deploy the service mesh in the operator namespace?

> I have tried it but it won't work because the ControlPlaneMode setting is on SMCP level. The easiest way to adjust this setting would be a configuration annotation like the current approach

But you can get the SMCP from the request, right?